cmd/restic/integration_test.go

```diff
@@ -436,6 +443,34 @@ func TestBackupExclude(t *testing.T) {
 		"expected file %q not in first snapshot, but it's included", "passwords.txt")
 }
 
+func TestBackupErrors(t *testing.T) {
+	if runtime.GOOS == "windows" {
```

Comment:
Why is this test skipped on windows?

Reply:
Enabling this test on windows would require creating a file which can not be read by restic. Since windows uses an ACL that is not POSIX-compliant, i don't see a straighforward way to implement this.

Reply:
Seems like the archiver also skips the test with unreadable files. So skipping the test is fine.